src/midiEditorCommands.cpp

```diff
@@ -202,13 +202,25 @@ void cmdMidiMoveCursor(Command* command) {
 		outputMessage(s);
 }
 
-const string getMidiNoteName(int pitch) {
+const string getMidiNoteName(MediaItem_Take *take, int pitch, int channel) {
 	static char* names[] = {"c", "c sharp", "d", "d sharp", "e", "f",
 		"f sharp", "g", "g sharp", "a", "a sharp", "b"};
-	int octave = pitch / 12 - 1;
-	pitch %= 12;
+	MediaTrack *track = GetMediaItemTake_Track(take);
+	int tracknumber = static_cast<int> (GetMediaTrackInfo_Value(track, "IP_TRACKNUMBER")); // one based
+	const char *note_name = GetTrackMIDINoteName(tracknumber - 1, pitch, channel); // track number is zero based
 	ostringstream s;
-	s << names[pitch] << " " << octave;
+	if (note_name) {
+		s << note_name;
+	} else {
+		int octave = pitch / 12 - 1;
+		int szOut = 0;
+		int *octave_offset = (int *)get_config_var("midioctoffs", &szOut);
```

Comment:
nit: Same as above for noteName. Also, just `(int*)` for the cast, no space before the asterisk.

Reply:
Changed